packages/react-components/src/components/DetailsCard/DetailsCard.module.scss

```diff
@@ -56,6 +51,27 @@ $base-class: 'details-card';
     }
   }
 
+  &__label {
+    display: flex;
+    align-items: center;
+    overflow: hidden;
+
+    &--with-margin {
+      margin-right: 36px;
```

Comment:
why not using spacing token?

Reply:
This is closely related to the behavior of the elements. In the case of the `hideLabelOnOpen` option, the chevron icon is a floating element that moves with the animation when the tab is opened. The value of this margin is not subject to spacing token because it is a space reserved for the described case, so that the text does not overlap the floating element.